AMBARI-23787 Log rotation for ambari metrics monitor log

majorendre opened the pull request:
## What changes were proposed in this pull request?

Log rotation added to ambari metrics monitor 

## How was this patch tested?

unit test and deployed to my cluster to check it

@swagle Please review

Commit:
AMBARI-23787 Log rotation for ambari metrics monitor log

asfgit commented:
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/Ambari-Github-PullRequest-Builder/2173/
Test FAILed.
Test FAILured.

swagle commented on ambari-metrics/ambari-metrics-host-monitoring/src/main/python/core/config_reader.py:
```diff
@@ -83,9 +83,9 @@ def get_ca_certs_file_path(self):
-OUT_DIR = os.path.join(os.sep, "var", "log", "ambari-metrics-host-monitoring")
-SERVER_OUT_FILE = OUT_DIR + os.sep + "ambari-metrics-host-monitoring.out"
-SERVER_LOG_FILE = OUT_DIR + os.sep + "ambari-metrics-host-monitoring.log"
+OUT_DIR = os.path.join(os.sep, "var", "log", "ambari-metrics-monitor")
```
Can this path be changed from the ini configuration file?

majorendre replied:
It is hardcoded right now.

Commit:
AMBARI-23787 log and out file config improvement

majorendre commented:
Added some config  handling for the log and out files.
Checked using the UI, changed "Metrics Monitor log dir", restarted, the files were opened at the specified directory.
@swagle Please review.

asfgit commented:
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/Ambari-Github-PullRequest-Builder/2219/
Test PASSed.
